### PR TITLE
polkit: upgrade to 0.115 and add elogind variant

### PR DESCRIPTION
**Pull request by Cogitri**
Please test this yourself, I can't test polkit right now. Polkit-elogind works wonderfully though.

### CODE REVIEW

**Comment by maxice8** [comment]
As expected `polkit` requires consolekit2 for auth and `polkit-elogind` needs elogind.

image
image

File: main/polkit-elogind/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Contributor: Carlo Landmeter
+# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
+pkgname=polkit
```
Comment (maxice8) [inline main/polkit-elogind/APKBUILD]: pkgname should be polkit-elogind

Comment (Cogitri) [reply main/polkit-elogind/APKBUILD]: Right, thank you!

**Comment by Cogitri** [comment]
Merged in ec563f54fcb69061dbbeb7ac0d4bc08455148f90

**Pull request closed without merging**